beam_test.py

```diff
@@ -175,12 +179,29 @@ def main():
             decode.decode_astropix2_hits(decode.hits_from_readoutstream(readout), i, file1)
             #decode.decode_astropix2_hits(decode.hits_from_readoutstream(readout))
             file1.write("\n")
+            
+            rows = [] #placeholder for list of rows hit
```

Comment:
@asteinhebel @Narg3000 you will need to change this line and the following line. The `plot_event` function takes two lists, the IDs of the triggering rows and columns. These can be lists or numpy arrays. Here I just put empty lists as placeholders.